core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -49,6 +52,7 @@ private[spark] class Slf4jSink(
   MetricsSystem.checkMinimalPollingPeriod(pollUnit, pollPeriod)
 
   val reporter: Slf4jReporter = Slf4jReporter.forRegistry(registry)
+    .outputTo(LoggerFactory.getLogger("org.apache.spark.metrics.sink.Slf4jSink"))
```

Comment:
Sure, it can be `classOf[Slf4jSink].getName` for simplicity and to avoid forgetting to update it if for some reason this moves.